crates/rpc/README.md

```diff
@@ -0,0 +1,31 @@
+# Silius RPC
+
+Silius RPC crate provides an interface for handling RPC methods according to the [EIP-4337 spec](https://eips.ethereum.org/EIPS/eip-4337#rpc-methods-eth-namespace).
+
+## Supported RPC Methods
+### `eth` name space
+* `eth_sendUserOperation`
+  * submits a User Operation object to the User Operation pool of the client. The client MUST validate the UserOperation, and return a result accordingly.
```

Comment:
Let's just unify how we reference User Operations... `User Operation object` or `UserOperation` or `user operation` ...

I prefer `UserOperation` since that's the struct name in the code. Similar to other types.